assets/global.js

```diff
@@ -398,6 +398,8 @@ class MenuDrawer extends HTMLElement {
     document.body.classList.remove(`overflow-hidden-${this.dataset.breakpoint}`);
     removeTrapFocus(elementToFocus);
     this.closeAnimation(this.mainDetailsToggle);
+
+    if (event instanceof KeyboardEvent) elementToFocus?.setAttribute('aria-expanded', false);
```

Comment:
Ok so [I found the source of the problem](https://github.com/Shopify/dawn/pull/2472/commits/df6fa0ce35702e797baf7938b08f4ebfafc50b15). I push a fix for it. I don't it's ideal in a way as it might not be very future proof and we could maybe have a better logic. But for now it does the trick. Let me know what you think 👍

Reply:
Yeah I'm imagining targeting those elements without using section-specific classes by checking if they don't have `details` parents or something, so it's completely generic, but I also don't know how feasible that is in practice, so what you have covers us for now.